mod_ci/models.py

```diff
@@ -34,9 +62,26 @@ class MaintenanceMode(Base):
     disabled = Column(Boolean, nullable=False, default=False)
 
     def __init__(self, platform, mode):
+        """
+        Parametrized constructor for the MaintenanceMode model
+
+        :param platform: The value of the 'platform' field of
+         MaintenanceMode model
+        :type platform: TestPlatform
+        :param disabled: mode
+        :type disabled: bool
+        """
         self.platform = platform
         self.disabled = mode
 
     def __repr__(self):
+        """
+        Representation function
+        Represent a MaintenanceMode Model by its platform Field.
+
+        :return platform, status: Returns the 'platform' and 'status'
```

Comment:
It doesn't return the fields, it returns a string representation of the object using the platform and status field values.